pkgs/applications/networking/sync/backintime/qt.nix

```diff
@@ -22,5 +22,27 @@ mkDerivation {
   preFixup = ''
     wrapQtApp "$out/bin/backintime-qt" \
       --prefix PATH : "${backintime-common}/bin:$PATH"
+
+    substituteInPlace "$out/share/polkit-1/actions/net.launchpad.backintime.policy" \
+      --replace "/usr/bin/backintime-qt" "$out/bin/backintime-qt"
+
+    substituteInPlace "$out/share/applications/backintime-qt-root.desktop" \
+      --replace "/usr/bin/backintime-qt" "backintime-qt"
+
+    substituteInPlace "$out/share/backintime/qt/serviceHelper.py" \
+      --replace "'which'" "'${which}/bin/which'" \
+      --replace "/bin/su" "${su}/bin/su" \
+      --replace "/usr/bin/backintime" "${backintime-common}/bin/backintime" \
+      --replace "/usr/bin/nice" "${coreutils}/bin/nice" \
+      --replace "/usr/bin/ionice" "${util-linux}/bin/ionice"
+
+    substituteInPlace "$out/share/dbus-1/system-services/net.launchpad.backintime.serviceHelper.service" \
+      --replace "/usr/bin/python3" "${python'}/bin/python3" \
+      --replace "/usr/share/backintime" "$out/share/backintime"
+
+    substituteInPlace "$out/bin/backintime-qt_polkit" \
+      --replace "/usr/bin/backintime-qt" "$out/bin/backintime-qt"
+    wrapProgram "$out/bin/backintime-qt_polkit" \
```

Comment:
I think a blank line above wrapProgram would be good here (and in line with the rest of the changes).

Reply:
fixed